[docs] Updates for Voyager 1.6.5 (#20897)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/macos.md

    The repository contains the formula to build and install `yb-voyager` on your macOS device.

>     Note that the tap `yugabyte/yugabytedb` has been updated to `yugabyte/tap`. if you have previously installed yb-voyager using the tap `yugabyte/yugabytedb`, untap the entry using `brew untap yugabyte/yugabytedb`, and then tap using the preceding command.

Reviewer: ```suggestion
    Note that the tap `yugabyte/yugabytedb` has been updated to `yugabyte/tap`. If you previously installed yb-voyager using the tap `yugabyte/yugabytedb`, untap the entry using `brew untap yugabyte/yugabytedb`, and then tap using the preceding command.
```

    ```sh
    brew install yb-voyager
    ```
    {{< note >}}

> Install `yb-voyager` with specific version on your machine using the following command:

Reviewer: ```suggestion
Install a specific version of `yb-voyager` using the following command:
```
